src/netlink.h

```diff
@@ -0,0 +1,37 @@
+#ifndef NETLINK_H
+#define NETLINK_H
+
+/*=========================================================================*\
+ * * NETLINK object
+ * * LuaSocket toolkit
+ * *
+ * * The netlink.h module provides LuaSocket with support for netlink protocol
+ * * (AF_NETLINK, SOCK_RAW).
+ * *
+ * * Two classes are defined: connected and unconnected. Netlink objects are
+ * * originally unconnected. They can be "connected" to a given address
+ * * with a call to the setpeername function. The same function can be used to
+ * * break the connection.
+\*=========================================================================*/
+
+#include "timeout.h"
+#include "lua.h"
+#include "socket.h"
+
+#define MAX_PAYLOAD 2048
+
+#define t_pid int
+#define t_groups int
+#define t_type int
```

Comment:
why defining this?

Reply:
Following the style used by Diego. But actually, it's supposed to be a typedef. Theres a similar section of code in usocket.h